lib/exercism/inbox.rb

```diff
@@ -38,6 +38,7 @@ def mark_as_read
   end
 
   def next_uuid(id)
+    id ||= user.exercises.first.id
```

Comment:
If there is no id, then we should default to `MAX(last_activity_at)` rather than a specific id, since we want the most recent in their inbox.